tests/suites/test_suite_ssl.function

```diff
@@ -1289,7 +1442,7 @@ void ssl_message_queue_overflow_underflow( )
 
     mbedtls_test_message_queue_setup( &queue, 3 );
 
-    /* 4 pushes (last with an error), 4 pops (last with an error) */
+    /* 4 pushes (last one with an error), 4 pops (last one with an error) */
```

Comment:
This should be changed in "Add a message metadata queue in ssl tests", not "Add a message-based socket mock connection to the ssl tests"